Add code to automatically disable inactive FreeIPA users (#73)
cisagov/ansible-role-freeipa-server: README.md

| Variable | Description | Default | Required |
|----------|-------------|---------|----------|
| freeipa_days_before_inactive | The number of days an account can go without being authenticated before it is determined to be inactive and is disabled. | `45` | No |

> | freeipa_disable_inactive_users | Whether or not to set up FreeIPA and create a systemd timer and service to disable inactive users. | `false` | No |

Reviewer: Minor complaint but I find the phrasing a bit ambiguous around the "set up FreeIPA" part. It would be nice to tie it more directly to the second half of the sentence by providing some mention of what "set up FreeIPA" entails.

Author: Please see commit 494db64.

| Variable | Description | Default | Required |
|----------|-------------|---------|----------|

> | freeipa_days_before_inactive | The number of days a user can go without logging in before his or her account is determined to be inactive and is disabled. | `45` | No |

Reviewer: Inclusivee language change. The comment in `defaults/main.yml` should also be updated.

```suggestion
| freeipa_days_before_inactive | The number of days a user can go without logging in before their account is determined to be inactive and is disabled. | `45` | No |
```

Author: Please see commit 206a290.

> | freeipa_disable_inactive_users | Whether or not to set up FreeIPA and systemd to periodically disable inactive users. | `false` | No |

Reviewer: Just to be consistent I think this should match the comment in `defaults/main.yml`.

```suggestion
| freeipa_disable_inactive_users | Whether or not to set up FreeIPA and create a systemd timer and service to disable inactive users. | `false` | No |
```
